### PR TITLE
Update sizing.md

### PR DESCRIPTION
**Pull request by jasnoyaeger**

### CODE REVIEW

File: docs/implementation-guide/sizing.md [hunk 1]
```diff
@@ -1,101 +1,98 @@
...
+- Minimum of 16GB RAM dedicated to VergeOS
+- HBA (preferred) or RAID controller that supports JBOD mode - NO RAID
+- 2 x 1/10/25/40GbE NICs (Intel, Mellanox, or Broadcom)
+
+!!! note "1GbE only recommended for External Network communication"
+
+### Controller Nodes (Node 1 and Node 2)
+
+- 1GB of RAM per 1TB of storage for each node
+- 1 x 800GB+ 3 DWPD NVMe-based SSD for vSAN metadata
```
Comment (lludlow) [inline docs/implementation-guide/sizing.md:18]: This should be a formula as noted below

File: docs/implementation-guide/sizing.md [hunk 2]
```diff
@@ -1,101 +1,98 @@
...
+- 5GB of vSAN metadata storage per 1TB of usable capacity in your VergeOS System
+
+### Storage Nodes (Nodes that partipate in vSAN storage)
+
+- 1GB of RAM per 1TB of storage for each node
```
Comment (lludlow) [inline docs/implementation-guide/sizing.md:23]: 1GB per 1TB raw storage

File: docs/implementation-guide/sizing.md [hunk 3]
```diff
@@ -1,101 +1,98 @@
...
-  - Compatible with any AMD, Intel, or Nvidia GPU.
-  - Nvidia vGPU also supported
+## Recommended Requirements
 
----
+### Generic Requirements (All Node Types) - Recommended
+
+- Intel or AMD x86 64-bit processor with hardware virtualization support
+- Minimum of 16GB RAM dedicated to VergeOS
+- HBA (preferred) or RAID controller that supports JBOD mode - NO RAID
```
Comment (lludlow) [inline docs/implementation-guide/sizing.md:51]: Ths is recommended requirements, I would not mention a RAID card here

File: docs/implementation-guide/sizing.md [hunk 4]
```diff
@@ -1,101 +1,98 @@
...
+- Minimum of 16GB RAM dedicated to VergeOS
+- 2 x 25/40GbE NICs (Intel, Mellanox, or Broadcom)
+- 2 x 10/25GbE NICs (Intel, Mellanox, or Broadcom)
+- **Dedicated Controller Nodes**
+
+### Controller Nodes (Node 1 and Node 2) - Recommended
 
-## Recomneded Requirements
+- 1 x +3.0Ghz CPU clockrate
```
Comment (lludlow) [inline docs/implementation-guide/sizing.md:59]: drop the word clockrate

File: docs/implementation-guide/sizing.md [hunk 5]
```diff
@@ -1,101 +1,98 @@
...
+- 2 x 800GB+ 3 DWPD NVMe-based SSD for vSAN metadata (Tier 0)
```
Comment (lludlow) [inline docs/implementation-guide/sizing.md:60]: see minimum requirements note, same should apply here

File: docs/implementation-guide/sizing.md [hunk 6]
```diff
@@ -1,101 +1,98 @@
...
-- **Compute Nodes**: Intel Xeon or AMD EPYC Processor, Dual 32 core Processors 2.2 GHz
-- Processor family must be consistent across the cluster
-- Must support VT-x (for Intel) or AMD-V/AMD-SVM (for AMD)
+### Storage Nodes (Nodes that partipate in vSAN storage) - Recommended
 
-### Memory
-- Minimum of 16 GB dedicated to VergeOS for OS and application server operations.
-- **Compute Nodes**: Memory to Core ratio of 8-1/16-1
-- **Storage Nodes**: 1.5GB per 1TB of RAW storage
+- 2 x +3.0Ghz CPU clockrate
```
Comment (lludlow) [inline docs/implementation-guide/sizing.md:65]: !clockrate

File: docs/implementation-guide/sizing.md [hunk 7]
```diff
@@ -1,101 +1,98 @@
...
+- 1 CPU core per disk
+- 1GB of RAM per 1TB of storage for each node
```
Comment (lludlow) [inline docs/implementation-guide/sizing.md:67]: being recommended, we would want more ram per raw TB of disk to allow for more caching 1.5 or 2gb per 1TB raw

File: docs/implementation-guide/sizing.md [hunk 8]
```diff
@@ -1,101 +1,98 @@
...
+
+The following table outlines the maximum supported hardware specifications for various resources in the VergeOS system as of version 4.12:
+
+| Resource                        | Maximum | Resource Type |
+|---------------------------------|---------|---------------|
+| Nodes per system                | 200     | node          |
+| Disks per VM [^3]               | 2000    | vdisk         |
+| Individual physical disk size   | 64      | terabyte      |
+| vDisk size                      | 2000    | terabyte      |
+| RAM per hos [^2]                | 5       | terabyte      |
```
Comment (lludlow) [inline docs/implementation-guide/sizing.md:91]: host

File: docs/implementation-guide/sizing.md [hunk 9]
```diff
@@ -1,101 +1,98 @@
...
+| Clusters per system             | 1000    | cluster       |
+| Tiers of storage per system     | 5       | tiers         |
+| vSAN Fault domains per system   | 2       | vSAN          |
+| Systems w/o Dedicated Controller Nodes | 10 | node |
```
Comment (lludlow) [inline docs/implementation-guide/sizing.md:95]: this is not clear, are we saying the max nodes in a system when the controllers participate in the vsan is 10?

File: docs/implementation-guide/sizing.md [hunk 10]
```diff
@@ -1,101 +1,98 @@
...
+- HBA (preferred) or RAID controller that supports JBOD mode - NO RAID
+- 2 x 1/10/25/40GbE NICs (Intel, Mellanox, or Broadcom)
+
+!!! note "1GbE only recommended for External Network communication"
+
+### Controller Nodes (Node 1 and Node 2)
+
+- 1GB of RAM per 1TB of storage for each node
+- 1 x 800GB+ 3 DWPD NVMe-based SSD for vSAN metadata
+- 5GB of vSAN metadata storage per 1TB of usable capacity in your VergeOS System
```
[the remark this answers is not in the document]
Comment (jasnoyaeger) [reply docs/implementation-guide/sizing.md:19]: same thing. Greg uses 5GB per 1TB of USABLE.

**Review by lludlow — APPROVED** [review]

**Pull request merged**